ocp_resources/node.py

```diff
@@ -1,16 +1,102 @@
+# Generated using https://github.com/RedHatQE/openshift-python-wrapper/blob/main/scripts/resource/README.md
+
+from typing import Any, Dict, Optional
 from ocp_resources.resource import Resource
 
 
 class Node(Resource):
     """
-    Node object, inherited from Resource.
+    Node is a worker node in Kubernetes. Each node will have a unique identifier
+    in the cache (i.e. in etcd).
+
+    API Link: https://docs.openshift.com/container-platform/4.16/rest_api/node_apis/node-v1.html
```

Comment:
this is a k8s resource; please add a link to the right api

Reply:
done